code/modules/holiday/easter.dm

```diff
@@ -165,19 +165,22 @@
 		qdel(src)
 
 //Easter Recipes + food
-/obj/item/reagent_containers/food/snacks/hotcrossbun
-	bitesize = 2
+/obj/item/food/hotcrossbun
+	bite_consumption = 2
 	name = "hot-cross bun"
 	desc = "The Cross represents the Assistants that died for your sins."
 	icon_state = "hotcrossbun"
+	foodtypes = SUGAR | GRAIN
+	tastes = list("easter", "fikou")
```

Comment:
@Fikou did you expect this one to sneak by

Reply:
i didnt do ti

Reply:
yeah fikou wtf. I think u should ban him for player references

Reply:
ok i did